iis-wf/iis-wf-primary/src/test/resources/eu/dnetlib/iis/wf/primary/processing/sampledataproducer/oozie_app/workflow.xml

```diff
@@ -426,6 +430,34 @@
 					<name>metric_pusher_address</name>
 					<value>${workingDir}/report/pushgateway</value>
 				</property>
+                <property>
+                    <name>patentMetadataRetrieverFacadeFactoryClassname</name>
+                    <value>eu.dnetlib.iis.wf.primary.processing.ClassPathBasedPatentServiceFacadeFactory</value>
+                </property>
+                <property>
+                    <name>patentServiceAuthnConsumerKey</name>
+                    <value>irrelevant</value>
+                </property>
+                <property>
+                    <name>patentServiceAuthnConsumerSecret</name>
+                    <value>irrelevant</value>
+                </property>
+                <property>
+                    <name>patentServiceEndpointAuthHost</name>
+                    <value>irrelevant</value>
+                </property>
+                <property>
+                    <name>patentServiceEndpointAuthUriRoot</name>
+                    <value>irrelevant</value>
+                </property>
+                <property>
+                    <name>patentServiceEndpointOpsHost</name>
+                    <value>irrelevant</value>
+                </property>
+                <property>
+                    <name>patentServiceEndpointOpsUriRoot</name>
+                    <value>irrelevant</value>
+                </property>
```

Comment:
I agree. This problem shows up not only in this particular test but also in many other cases where we use mocks due to connection with remote services. I don't think we are verifying whether the unused (in the test) parameter value is properly propagated.

This set of properties is being used only by the `OpenPatentWebServiceFacadeFactory` which is defined as `patentMetadataRetrieverFacadeFactoryClassname` wf parameter while the `PatentMetadataRetrieverJob` is pretty generic in terms of what patent service facade will be used. So if we are not testing `OpenPatentWebServiceFacadeFactory`, which is the case, then we might not want to check the params which are unused.

But... since we are living in oozie world and I have defined those properties in wf definition then we should treat them as mandatory even if the patent service facade used in the test does not require them. It is part of the contract between patent wf and uber wf. So in order to be on a safe side I have supplemented `ClassPathBasedPatentServiceFacadeFactory` with simple parameter values verification.

Reply:
Yes, I agree with your point but I think `ClassPathBasedPatentServiceFacadeFactory` was not changed. Maybe missing commit ?

Reply:
OK, in this particular case I can see the changes were committed and pushed:

https://github.com/openaire/iis/pull/1091/commits/e3c3d1e30abffdff3081aa033b69b0e3613ad21e#diff-3a25b33e863f4422043635d45b7d880b

We need to look into our code-review cycle and verify what makes those changes not appearing on your end.